src/templates/Header/Header.tsx

```diff
@@ -0,0 +1,66 @@
+import { Grid, Link, Typography, useMediaQuery } from '@mui/material';
+import styles from './Header.module.scss';
+import { useTheme } from '@mui/material/styles';
```

Comment:
I suggest creating common interface 
```typescript
interface NamedUrl {
  name: string;
  url: string;
}
```
that can be used for facultyName + url universityName + url ....

Reply:
resolved in f29528d